modules/ocf_printhost/files/enforcer

```diff
@@ -117,6 +117,15 @@ NOTIFY_JOB_ERROR = dedent("""\
         Please contact a staff member for assistance.\
 """)
 
+NOTIFY_PROHIBITED_FILE_TYPE = dedent("""\
+        Unfortunately we don't support printing {file_type}.
```

Comment:
I'd just say "We've encountered problems printing documents from this source. Please try to download your document as a PDF and print it from the desktop instead of the web browser." or something like that